src/index.js

```diff
@@ -213,3 +213,9 @@ export OrderedList from 'carbon-components-react/lib/components/OrderedList';
 
 export TabContent from 'carbon-components-react/lib/components/TabContent';
 export UnorderedList from 'carbon-components-react/lib/components/UnorderedList';
+
+// Unstable experimental components
+export {
+  PageSelector,
+  UNSTABLE_Pagination,
```

Comment:
Would you mind pointing to Carbon's naming convention for this? I couldn't find it on a cursory search.